src/cf-typography/src/atoms/micro-copy.less

```diff
@@ -0,0 +1,8 @@
+.a-micro-copy {
+    color: @micro-copy;
+    font-size: unit( @size-v / @base-font-size-px, em );
+
+    &__large {
+        font-size: unit( @base-font-size-px / @base-font-size-px, em );
+    }
```

Comment:
I searched and found those, too, but I still don't understand what the point is. It just ends up being the default body text of 16px Avenir. I vote to kill it.